src/ert/callbacks.py

```diff
@@ -59,6 +61,7 @@ def _write_responses_to_storage(
                 f"Loaded {config.name}",
                 extra={"Time": f"{(time.perf_counter() - start_time):.4f}s"},
             )
+            await asyncio.sleep(0)
```

Comment:
I guess (all these) await asyncio.sleep(0) is for the sake of yielding right?

Reply:
yep!